sigs.yaml

```diff
@@ -1611,13 +1616,19 @@ sigs:
       - name: sig-scalability-test-failures
         description: Test Failures and Triage
     subprojects:
-    - name: kubemark
+    - name: kubernetes-scalability-definition
```

Comment:
What about other things under sig-scalability/?
Like configs&limits, processes, etc?

Reply:
That was supposed to be very initial version as I don't have good name for it.
I would like to extend it a bit more (in particular add more descriptions etc., which I wrote some time ago when I was working on charter (but then the template for charter changed).
But i wanted to push have it in sooner.

Does that sound ok for you?

Reply:
Yeah, that sgtm.